__tests__/Unit/hooks/taskDetailsApi.test.tsx

```diff
@@ -17,7 +17,7 @@ beforeAll(() => {
 });
 afterEach(() => server.resetHandlers());
 afterAll(() => server.close());
-const taskIds = ['6KhcLU3yr45dzjQIVm0J', '6KhcLU3yr45dzjQIVmod'];
+const taskIds = ['6KhcLU3yr45dzjQIVm0J'];
```

Comment:
Why are we changing this?

Reply:
in case of it's presence, the second ID is getting passed inside a test, but only the first one should be passed and after this removal things are working fine 👍🏻